cmd/check.go

```diff
@@ -14,7 +15,8 @@ func init() {
 		Short: "Check for out-of-date dependencies",
 		Long:  "Scans through direct and indirect dependencies to check if a newer version is available. Exceptions can be set by 'modtools freeze'",
 		RunE: func(cmd *coral.Command, args []string) error {
-			res, err := checkDeps()
+			err := rootCmd.Flags().Parse(os.Args)
```

Comment:
This value of `err` is never used. Also, I think this is not needed at all as coral parses the flags automatically.

Reply:
yeah that is true, I was trying  to figure out why something wasn't working and then didn't drop this line :facepalm: